tests/exception_handling_cases.py

```diff
@@ -0,0 +1,64 @@
+# flake8: noqa
+import contextlib
+
```

Comment:
Make sure you have a test with nested try-excepts such that there is code inside the outer try both before and after the inner try (and it gets two entries in the exception table).

Reply:
Thanks ! I will make sure to add such a case.